{"name": "test_invalid_sign_in_put_request", "status": "passed", "steps": [{"name": "Fail sign in put request", "status": "passed", "start": 1648801687521, "stop": 1648801687730}], "start": 1648801687520, "stop": 1648801687730, "uuid": "8367fe98-c046-429a-b84a-e1795a980025", "historyId": "57c3d0ec224a396623f35ab63918ea3b", "testCaseId": "f1d4115cb05828fcbaa8a2d8e3f9c480", "fullName": "tests.api_testcases.login_test.login_test.TestUserLogin#test_invalid_sign_in_put_request", "labels": [{"name": "parentSuite", "value": "tests.api_testcases.login_test"}, {"name": "suite", "value": "login_test"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "pop-os"}, {"name": "thread", "value": "119077-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api_testcases.login_test.login_test"}]}